src/test/java/com/jfrog/ide/common/configuration/JfrogCliDriverTest.java

```diff
@@ -62,27 +66,30 @@ private void configJfrogCli() {
             tempDir = Files.createTempDirectory("ide-plugins-common-cli-test").toFile();
             tempDir.deleteOnExit();
             getCli(tempDir);
-        } catch (IOException e) {
+        } catch (IOException | InterruptedException e) {
             fail(e.getMessage(), e);
         }
         testEnv.put("JFROG_CLI_HOME_DIR", tempDir.getAbsolutePath());
         jfrogCliDriver = new JfrogCliDriver(testEnv, tempDir.getAbsolutePath() + File.separator);
 
     }
 
-    private void getCli(File execDir) throws IOException {
+    private void getCli(File execDir) throws IOException, InterruptedException {
         List<String> args;
         if (SystemUtils.IS_OS_WINDOWS) {
-            args = Lists.newArrayList("cmd", "/c", "curl -XGET" +
-                    " \"https://releases.jfrog.io/artifactory/jfrog-cli/v2/[RELEASE]/jfrog-cli-windows-amd64/jfrog.exe\" -L -k -g");
-        } else {
-            args = new ArrayList<>() {{
-                add("/bin/sh");
-                add("-c");
-                add("curl -fL https://getcli.jfrog.io | bash -s v2");
-            }};
+            InputStream in = new URL("https://releases.jfrog.io/artifactory/jfrog-cli/jf-v2/[RELEASE]/jfrog-cli-windows-amd64/jf.exe").openStream();
+            Files.copy(in, Paths.get(tempDir.getAbsolutePath() + "\\jf.exe"), StandardCopyOption.REPLACE_EXISTING);
+            return;
         }
+
+        args = new ArrayList<>() {{
+            add("/bin/sh");
+            add("-c");
+            add("curl -fL https://getcli.jfrog.io/jf-v2 | sh");
+        }};
+
         Process process = Runtime.getRuntime().exec(args.toArray(new String[0]), new String[0], execDir);
+        process.waitFor();
         process.getOutputStream().close();
```

Comment:
Let's use "try with resources" to close this, to make sure it is always closed at the end.